prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -26,10 +27,12 @@ import java.time.format.DateTimeParseException
  * Utilities to handle FHIR Path parsing.
  */
 object FhirPathUtils : Logging {
+
+    private val fhirContext = FhirContext.forR4()
     /**
      * The FHIR path engine.
      */
-    val pathEngine = FHIRPathEngine(SimpleWorkerContext())
+    val pathEngine = FHIRPathEngine(HapiWorkerContext(fhirContext, fhirContext.validationSupport))
```

Comment:
The reason for this change is interesting.  [This](https://github.com/hapifhir/org.hl7.fhir.core/commit/26027fa3d6d777fb46abc4e0e17b1f4cd04d6068#diff-9412a76f9f92124fb0838f3589aba8264ce989c0c400701f82ba6d16f822fa11R1805) PR introduced the `isKnownType` function which calls `worker.fetchTypeDefintion` which looks in the `structures`; this value is empty for the SimpleWorkerContext.

I actually think this is technically the correct worker to use anyway